### PR TITLE
Update few sysctl rules to accept multiple compliant values

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Update the rules listed below to accept multiple values as compliant.
  - `sysctl_net_ipv4_conf_all_rp_filter`
  - `sysctl_kernel_kptr_restrict`

#### Rationale:

- Better align the rules with the RHEL and OL STIGs.

### CODE REVIEW

File: linux_os/guide/system/network/network-kernel/network_host_and_router_parameters/sysctl_net_ipv4_conf_all_rp_filter/rule.yml [hunk 1]
```diff
@@ -59,4 +59,10 @@ template:
     name: sysctl
     vars:
         sysctlvar: net.ipv4.conf.all.rp_filter
+        {{% if 'ol' in product or 'rhel' in product %}}
+        sysctlval:
+        - '1'
+        - '2'
```
Comment (jan-cerny) [inline linux_os/guide/system/network/network-kernel/network_host_and_router_parameters/sysctl_net_ipv4_conf_all_rp_filter/rule.yml:65]: I think that the description and OCIL should be updated to describe that both 1 and 2 values are acceptable.

**Review by jan-cerny — COMMENTED** [review]
The CI fail is probably caused by the container environment, when I execute it on a virtual machine backend, they all pass:
```
[jcerny@thinkpad scap-security-guide{pr/9283}]$ python3 tests/automatus.py rule --libvirt qemu:///system ssgts_rhel9 sysctl_kernel_kptr_restrict,sysctl_net_ipv4_conf_all_rp_filter
Setting console output to log level INFO
INFO - The base image option has not been specified, choosing libvirt-based test environment.
INFO - Logging into /home/jcerny/work/git/scap-security-guide/logs/rule-custom-2022-08-03-1315/test_suite.log
INFO - xccdf_org.ssgproject.content_rule_sysctl_net_ipv4_conf_all_rp_filter
INFO - Script two_sysctls_on_same_file_name.fail.sh using profile (all) OK
INFO - Script symlinks_to_same_file.pass.sh using profile (all) OK
INFO - Script line_not_there.fail.sh using profile (all) OK
INFO - Script wrong_value.fail.sh using profile (all) OK
INFO - Script comment.fail.sh using profile (all) OK
INFO - Script two_sysctls_on_same_file.pass.sh using profile (all) OK
INFO - Script symlink_repeated_sysctl_conf.pass.sh using profile (all) OK
INFO - Script symlink_same_option.fail.sh using profile (all) OK
INFO - Script two_sysctls_on_d.fail.sh using profile (all) OK
INFO - Script symlink_root_incompliant.fail.sh using profile (all) OK
INFO - Script wrong_runtime.fail.sh using profile (all) OK
INFO - Script one_sysctl_conf_one_sysctl_d.fail.sh using profile (all) OK
INFO - Script symlink_different_option.pass.sh using profile (all) OK
INFO - Script wrong_value_d_directory.fail.sh using profile (all) OK
INFO - Script correct_value.pass.sh using profile (all) OK
INFO - Script symlink_root_duplicate.fail.sh using profile (all) OK
INFO - Script value_1.pass.sh using profile (all) OK
INFO - Script value_2.pass.sh using profile (all) OK
INFO - xccdf_org.ssgproject.content_rule_sysctl_kernel_kptr_restrict
INFO - Script value_1.pass.sh using profile (all) OK
INFO - Script value_2.pass.sh using profile (all) OK
[jcerny@thinkpad scap-security-guide{pr/9286}]$ python3 tests/automatus.py rule --libvirt qemu:///system ssgts_rhel9 --remediate-using ansible sysctl_kernel_kptr_restrict,sysctl_net_ipv4_conf_all_rp_filter
Setting console output to log level INFO
INFO - The base image option has not been specified, choosing libvirt-based test environment.
INFO - Logging into /home/jcerny/work/git/scap-security-guide/logs/rule-custom-2022-08-03-1321/test_suite.log
INFO - xccdf_org.ssgproject.content_rule_sysctl_net_ipv4_conf_all_rp_filter
INFO - Script one_sysctl_conf_one_sysctl_d.fail.sh using profile (all) OK
INFO - Script symlink_root_duplicate.fail.sh using profile (all) OK
INFO - Script two_sysctls_on_d.fail.sh using profile (all) OK
INFO - Script two_sysctls_on_same_file.pass.sh using profile (all) OK
INFO - Script symlink_repeated_sysctl_conf.pass.sh using profile (all) OK
INFO - Script symlink_same_option.fail.sh using profile (all) OK
INFO - Script correct_value.pass.sh using profile (all) OK
INFO - Script symlink_root_incompliant.fail.sh using profile (all) OK
INFO - Script wrong_runtime.fail.sh using profile (all) OK
INFO - Script wrong_value.fail.sh using profile (all) OK
INFO - Script symlink_different_option.pass.sh using profile (all) OK
INFO - Script two_sysctls_on_same_file_name.fail.sh using profile (all) OK
INFO - Script line_not_there.fail.sh using profile (all) OK
INFO - Script comment.fail.sh using profile (all) OK
INFO - Script symlinks_to_same_file.pass.sh using profile (all) OK
INFO - Script wrong_value_d_directory.fail.sh using profile (all) OK
INFO - Script value_1.pass.sh using profile (all) OK
INFO - Script value_2.pass.sh using profile (all) OK
INFO - xccdf_org.ssgproject.content_rule_sysctl_kernel_kptr_restrict
INFO - Script value_1.pass.sh using profile (all) OK
INFO - Script value_2.pass.sh using profile (all) OK
```

File: linux_os/guide/system/network/network-kernel/network_host_and_router_parameters/sysctl_net_ipv4_conf_all_rp_filter/rule.yml (hunk 1 above)
Comment (yuumasato) [reply linux_os/guide/system/network/network-kernel/network_host_and_router_parameters/sysctl_net_ipv4_conf_all_rp_filter/rule.yml:65]: Good catch!

Comment (yuumasato) [reply linux_os/guide/system/network/network-kernel/network_host_and_router_parameters/sysctl_net_ipv4_conf_all_rp_filter/rule.yml:90]: Updated based on your recent PR.

**Review by jan-cerny — APPROVED** [review]
```
[jcerny@thinkpad scap-security-guide{pr/9286}]$ python3 tests/automatus.py rule --libvirt qemu:///system ssgts_rhel9 sysctl_kernel_kptr_restrict
Setting console output to log level INFO
INFO - The base image option has not been specified, choosing libvirt-based test environment.
INFO - Logging into /home/jcerny/work/git/scap-security-guide/logs/rule-custom-2022-08-04-1402/test_suite.log
INFO - xccdf_org.ssgproject.content_rule_sysctl_kernel_kptr_restrict
INFO - Script one_sysctl_conf_one_sysctl_d.fail.sh using profile (all) OK
INFO - Script symlink_repeated_sysctl_conf.pass.sh using profile (all) OK
INFO - Script symlink_same_option.fail.sh using profile (all) OK
INFO - Script symlink_root_incompliant.fail.sh using profile (all) OK
INFO - Script two_sysctls_on_same_file.pass.sh using profile (all) OK
INFO - Script line_not_there.fail.sh using profile (all) OK
INFO - Script symlink_root_duplicate.fail.sh using profile (all) OK
INFO - Script symlinks_to_same_file.pass.sh using profile (all) OK
INFO - Script comment.fail.sh using profile (all) OK
INFO - Script two_sysctls_on_d.fail.sh using profile (all) OK
INFO - Script wrong_runtime.fail.sh using profile (all) OK
INFO - Script two_sysctls_on_same_file_name.fail.sh using profile (all) OK
INFO - Script correct_value.pass.sh using profile (all) OK
INFO - Script wrong_value_d_directory.fail.sh using profile (all) OK
INFO - Script symlink_different_option.pass.sh using profile (all) OK
INFO - Script wrong_value.fail.sh using profile (all) OK
INFO - Script value_1.pass.sh using profile (all) OK
INFO - Script value_2.pass.sh using profile (all) OK
[jcerny@thinkpad scap-security-guide{pr/9286}]$ python3 tests/automatus.py rule --libvirt qemu:///system ssgts_rhel9 sysctl_net_ipv4_conf_all_rp_filter
Setting console output to log level INFO
INFO - The base image option has not been specified, choosing libvirt-based test environment.
INFO - Logging into /home/jcerny/work/git/scap-security-guide/logs/rule-custom-2022-08-04-1408/test_suite.log
INFO - xccdf_org.ssgproject.content_rule_sysctl_net_ipv4_conf_all_rp_filter
INFO - Script two_sysctls_on_same_file.pass.sh using profile (all) OK
INFO - Script one_sysctl_conf_one_sysctl_d.fail.sh using profile (all) OK
INFO - Script correct_value.pass.sh using profile (all) OK
INFO - Script symlink_repeated_sysctl_conf.pass.sh using profile (all) OK
INFO - Script wrong_runtime.fail.sh using profile (all) OK
INFO - Script symlink_root_duplicate.fail.sh using profile (all) OK
INFO - Script two_sysctls_on_d.fail.sh using profile (all) OK
INFO - Script comment.fail.sh using profile (all) OK
INFO - Script symlink_same_option.fail.sh using profile (all) OK
INFO - Script symlink_root_incompliant.fail.sh using profile (all) OK
INFO - Script two_sysctls_on_same_file_name.fail.sh using profile (all) OK
INFO - Script wrong_value.fail.sh using profile (all) OK
INFO - Script line_not_there.fail.sh using profile (all) OK
INFO - Script symlinks_to_same_file.pass.sh using profile (all) OK
INFO - Script symlink_different_option.pass.sh using profile (all) OK
INFO - Script wrong_value_d_directory.fail.sh using profile (all) OK
INFO - Script value_1.pass.sh using profile (all) OK
INFO - Script value_2.pass.sh using profile (all) OK
[jcerny@thinkpad scap-security-guide{pr/9286}]$ 
```

**Comment by ggbecker** [comment]
Unfortunately there are issues with the build using OVAL 5.10:

```
Traceback (most recent call last):
  File "/home/ggasparb/workspace/github/content/build-scripts/relabel_ids.py", line 65, in <module>
    main()
  File "/home/ggasparb/workspace/github/content/build-scripts/relabel_ids.py", line 52, in main
    oval_linker.link_xccdf()
  File "/home/ggasparb/workspace/github/content/ssg/build_renumber.py", line 93, in link_xccdf
    self.add_missing_check_exports(check, checkcontentref)
  File "/home/ggasparb/workspace/github/content/ssg/build_renumber.py", line 221, in add_missing_check_exports
    for def_id in self.get_nested_definitions(check_name):
  File "/home/ggasparb/workspace/github/content/ssg/build_renumber.py", line 207, in get_nested_definitions
    extensions = find_extending_defs(self.oval_groups, definition_tree)
  File "/home/ggasparb/workspace/github/content/ssg/parse_oval.py", line 98, in find_extending_defs
    return _find_attr(oval_groups, defn, "extend_definition", "definition_ref")
  File "/home/ggasparb/workspace/github/content/ssg/parse_oval.py", line 89, in _find_attr
    finder.find_element(defn, elem, attr)
  File "/home/ggasparb/workspace/github/content/ssg/parse_oval.py", line 35, in find_element
    self._recurse(start_element)
  File "/home/ggasparb/workspace/github/content/ssg/parse_oval.py", line 44, in _recurse
    self._recurse(child)
  File "/home/ggasparb/workspace/github/content/ssg/parse_oval.py", line 44, in _recurse
    self._recurse(child)
  File "/home/ggasparb/workspace/github/content/ssg/parse_oval.py", line 42, in _recurse
    self._examine_element(element)
  File "/home/ggasparb/workspace/github/content/ssg/parse_oval.py", line 61, in _examine_element
    self._recurse(new_root)
  File "/home/ggasparb/workspace/github/content/ssg/parse_oval.py", line 44, in _recurse
    self._recurse(child)
  File "/home/ggasparb/workspace/github/content/ssg/parse_oval.py", line 42, in _recurse
    self._examine_element(element)
  File "/home/ggasparb/workspace/github/content/ssg/parse_oval.py", line 58, in _examine_element
    new_root = self.oval_groups[reference_target][entity_id]
KeyError: 'object_sysctl_kernel_unprivileged_bpf_disabled_accept_default_static_set_sysctls'
[10/20] [rhel9-content] generating ssg-rhel9-cpe-dictionary.xml, ssg-rhel9-cpe-oval.xml
ninja: build stopped: subcommand failed.
```

More code here should be applicable only to OVAL 5.11:
https://github.com/ComplianceAsCode/content/blob/master/shared/templates/sysctl/oval.template#L156-L186

**Pull request merged**